python/consumer/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+USER root
+
+RUN microdnf update \
+    && microdnf install python3-devel shadow-utils \
```

Comment:
I have inimal experience with Python ... is this a common way to build a Python application image to install `python-devel` and copy the dependencies with `pip install` inside the image?

Reply:
Looking into this

Reply:
I've switched to use `/ubi8/python-38:latest` as recommended by Mickael, and that removed the need to run `microdnf install python3-devel` inside the image.